azuredeploy.json

```diff
@@ -9,6 +9,30 @@
                 "description": "Prefix for all resources created by this template"
             }
         },
+        "location": {
+            "type": "string",
+            "defaultValue": "South Central US",
+            "allowedValues": [
+                "East Asia",
+                "Southeast Asia",
+                "Australia East",
+                "Brazil South",
+                "Canada Central",
+                "North Europe",
+                "West Europe",
+                "Central India",
+                "UK South",
+                "East US",
+                "East US 2",
+                "South Central US",
+                "West Central US",
+                "West US",
+                "West US 2"
+            ],
+            "metadata": {
+                "description": "Location where all resources will be deployed."
```

Comment:
Re-iterating, I'd like for this to be a restriction not on *all* resources, but only search services?
Not a big deal, just wondering what the issue would be in such a case

Reply:
There's no issue, the demo will just likely be more efficient if all of the resources are in the same region (custom skills will be closer to the search service).  But its easy to change back if only restricting the search service is preferred.